### PR TITLE
Fix chat colors and username display

### LINKED ISSUE #6979: Update the colours and username display inside the mobile chat UI
# Problem

As part of the ongoing work on UI components we've made some changes to the mobile UI that aim to unify it across all screens and products, this issue is related to introducing those changes into chat message UI.

# Implementation

comparison

### The scope of this task is related to two areas:
_Change of chat background from grey to white, and incoming messages from white to light blue, with some minor colour tweaks that go along with it. The purpose of that is unifying the visual design across all screens and mobile-desktop chat UIs_

- [ ] Change the chat background colour to white (#FFF)
- [ ] Change the Incoming message bubble background colour to Light Blue (#ECEFFC)
- [ ] Outgoing reply message, change the colour to 70% white (rgba 255,255,255, 0.7) of following:
- the quoted username colour 
- quoted text colour 
- the timestamp colour
 / _this is a fix, currently, the contrast ratio is too low there to be readable_
Incoming reply message, the 2px high underline below the quoted snippet should be 10% black (rgba 0,0,0 0.1)

_Change the way we render usernames inside chat to better distinguish between chosen username and 3w anonymous username_

- [ ] Chosen username, font-size 13px font-weight: 500, color: Dark Grey (#939BA1), line-height: 16px
- [ ] 3w anonymous name, font-size: 12px, font-weight 400, color: Dark Grey (#939BA1), line-height: 16px
- [x] change the :: characters to a bullet • (opt+8 on mac)
- [ ] 12px padding on the left, 16px on the right (the username begins at the same place as text inside message bubbles)
- [ ] the entire username should span the entire screen width and not only the message bubble (it can be wider than the message bubble)
- inside message replies, the entire quoted username should be truncated with ellipsis (…) if it overflows and doesn't fit in a line

# Acceptance Criteria

screenshot 2018-12-03 at 11 21 16

All improvements were introduced as outlined [in this design](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=1254%3A1791) 

thanks a bunch!

### PR DESCRIPTION
**Pull request by vitvly**
Fixes #6979

### CODE REVIEW

File: src/status_im/ui/screens/chat/styles/message/message.cljs
Comment (jeluard) [inline src/status_im/ui/screens/chat/styles/message/message.cljs]: Could be clearer with a cond?

File: src/status_im/ui/screens/chat/styles/message/message.cljs
Comment (jeluard) [inline src/status_im/ui/screens/chat/styles/message/message.cljs:183]: Better to align map values

**Review by jeluard — APPROVED** [review]

**Comment by errorists** [comment]
@siphiuel Hey, some issues I've spotted: 
The formatting on incoming replies looks incorrect, the line below quoted reply is grey, should be 10% black. 
There is a weird white duplicate timestamp in each message 
some of the margins need fixes 
thanks man! if you need any assistance be sure to msg me :)

screenshot

**Review by yenda — APPROVED** [review]

**Comment by rachelhamlin** [comment]
Hey @siphiuel, are you making the fixes that @errorists pointed out before merging #7304?

**Comment by vitvly** [comment]
@rachelhamlin yes, I've switched to TtT screens in the meanwhile, so didn't push the changes yet here. We've been communicating about it with @errorists

**Comment by vitvly** [comment]
@errorists finally fixed the margins, ingoing reply line color, and duplicate timestamp thing.

File: src/status_im/ui/screens/chat/styles/message/message.cljs
Comment (vitvly) [reply src/status_im/ui/screens/chat/styles/message/message.cljs]: Done.

File: src/status_im/ui/screens/chat/styles/message/message.cljs
Comment (vitvly) [reply src/status_im/ui/screens/chat/styles/message/message.cljs:183]: Aligned.

**Comment by errorists** [comment]
> @errorists finally fixed the margins, ingoing reply line color, and duplicate timestamp thing.

@siphiuel Thanks! 🙏 There's a new issue now, timestamps in outgoing messages are invisible, I guess the white duplicates you fixed were a part of outgoing messages. I also didn't have a chance to test transaction messages thoroughly.

**Comment by vitvly** [comment]
@errorists ahh right, fixed the outgoing messages timestamp color (it's 70% white now). Thanks!

**Comment by churik** [comment]
@siphiuel can't be built because file is formatted incorrectly. Can you please fix it?
```
(defn message-timestamp-text [justify-timestamp? outgoing rtl?]

   (merge message-timestamp

-         {:color (if outgoing 

+         {:color (if outgoing

                    (colors/alpha colors/white 0.7)

                    colors/gray)}

          (when justify-timestamp? {:position              :absolute

1 file(s) formatted incorrectly

script returned exit code 1
```

**Comment by vitvly** [comment]
@churik fixed broken formatting

**Comment by churik** [comment]
@siphiuel  following issues should be fixed:
ready
ready
ready3

**(1, 4, 5) No visible border between chat header, menus  and message area - all reproducible for IOS, for Android only 4 can be replicated
(2) No visible currency for incoming `/send ` message
(3) White screen inside incoming `/request` message**

Not sure that designs are updated for these elements.


<blockquote></blockquote>

**Comment by errorists** [comment]
thanks @churik!
1) yes, there should be a 1px Light Grey border on the bottom of the Top Bar - it's a wider issue shared with most of the screens in the app. All designs in Figma reflect that as well as our components, however we didn't file an issue for fixing that as we believe it's best resolved on global component level, either the current toolbar component needs updating or it needs to be replaced with a new one cc @yenda 
2) thanks for spotting, the currency string should be using our Grey colour [Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=3%3A556)
3) the white part looks weird, it shouldn't have a background at all
4) The 1px Light Grey bottom divider is missing on the 'add to contacts' snackbar [Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=2%3A126)
5) The autocomplete box is missing a shadow `box-shadow: 0px 1px 12px rgba(0, 0, 0, 0.12)` but if it's easier to add then a 1px Light Grey divider on top will also suffice. [Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=144%3A237)

Most of the above are long overdue issues even before the colour change, if we can update them accordingly it would leave the mobile Chat UI in a much-improved shape. All Figma designs reflect that, lmk if I can be of further help.

**Comment by vitvly** [comment]
Items 2,3,4 fixed. For item 5 i've added a light-grey top border, hopefully this will work (did not test on ios)

**Comment by vitvly** [comment]
Item 1 - I can see the border on Android:
https://monosnap.com/file/h8xQLQqbmtkrENJEmkMNFNdYofukrp

<blockquote><div>Monosnap screenshot tool</div><div><strong><a href="https://monosnap.com/file/h8xQLQqbmtkrENJEmkMNFNdYofukrp">Screenshot "Android Emulator - Pixel_2_API_275554 2019-02-05 17-48-23.png"</a></strong></div></blockquote>

**Comment by churik** [comment]
sdfsdffddfsdfdfsdf


1) @siphiuel  white dot after amount and before currency  (Android only) - shouldn't be here

2) @errorists about  1 - timestamps should be presented for outgoing emojis, yes? They were hardly visible (https://github.com/status-im/status-react/issues/6497); they are not  visible at all after changing background color.
Will be fixed separately or should be here, WDYT?

3)  @errorists elements are not centered inside icons - but I suppose not related to PR, should be reported separately?

Other issues are fixed.

**Comment by errorists** [comment]
@churik once again, thanks a bunch for testing 👏 

2) Definitely, they should. correct colour is Grey, similar to incoming emojis. Also it would be nice if the timestamp was placed in the same baseline as emoji, see [this Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=2%3A541)
I'd say to fix it here in this issue, once pushed we'll have a solid clean chat UI to build further features on
3) there's a couple of things going wrong there, I suspect it's actually caused by the recent PR where Eric updated all icon resources. The text is aligned correctly but the icon is too large and overflows the container, definitely needs to be fixed, I leave it to your judgement when. The correct icon resource should be the `tiny:check` icon, [link to Figma](https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=787%3A1024), and the `tiny pending` icon needs to be added, [link to Figma](https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=1875%3A1797)
For reference, the various transaction dialogues can be found here [Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=2%3A527)

**Comment by vitvly** [comment]
@churik @errorists 
1. Fixed
2. Changed timestamp color to gray and moved it to the same line as the emoji.
3. Added a tiny-pending icon and used both tiny-check and tiny-pending in send-preview.

**Comment by errorists** [comment]
@siphiuel you're a legend, thanks! 🙏  one last thing which Eric's PR bugged, the icon for quote bubbles when you're replying to someone, it's not using the `tiny-reply` asset. If you guys recall any other chat specific icons that might have been bugged, pls don't hesitate to fix them, honestly it feels like we're putting down fires caused by his PR xD

**Comment by yenda** [comment]
@errorists unfortunately the old icons where not sorted and never named tiny so I missed out on that :(

**Comment by vitvly** [comment]
Changed to use tiny-reply instead of main-reply.

**Comment by errorists** [comment]
🍾 woohoo, I think this PR is done and ready to be merged friends 👏

**Comment by rachelhamlin** [comment]
Nice work! 🤸‍♀️

File: src/status_im/ui/components/icons/vector_icons.cljs
Comment (yenda) [inline src/status_im/ui/components/icons/vector_icons.cljs]: please alphabetical order and also import for desktop as well just in case

Comment (vitvly) [reply src/status_im/ui/components/icons/vector_icons.cljs]: Done.

**Pull request merged**
